xbmc/addons/gui/GUIDialogAddonInfo.cpp

```diff
@@ -153,8 +153,8 @@ bool CGUIDialogAddonInfo::OnAction(const CAction& action)
 
 void CGUIDialogAddonInfo::OnInitWindow()
 {
-  UpdateControls();
   CGUIDialog::OnInitWindow();
+  UpdateControls();
 }
```

Comment:
the order of these function calls needed to be switched for button focussing working correctly.
https://github.com/xbmc/xbmc/commit/a3b58aae6ae3c09529cdc08f097bef8c7e388599 changed this part about 10 yrs. ago, so button focussing might need special attention and testing now